src/nextcloud/utilities/nextcloud-utilities

```diff
@@ -18,3 +19,31 @@ wait_for_nextcloud_to_be_configured()
 		printf "done\n"
 	fi
 }
+
+cronjob_interval()
+{
+	interval="$(snapctl get nextcloud.cron-interval)"
+	if [ -z "$interval" ]; then
+		interval="$DEFAULT_CRONJOB_INTERVAL"
+		set_cronjob_interval "$interval"
+	fi
+
+	set_previous_cronjob_interval "$interval"
```

Comment:
This needs to be under the `if`. Please refer to other configurable items for how this is done, for example [apache](https://github.com/nextcloud/nextcloud-snap/blob/master/src/apache/utilities/apache-utilities#L61).